### PR TITLE
Amplitude stats blog

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

* Amplitude stats blog
* Amplitude contributor training blog

Current usage rendered at https://20241103-amplitude-stats.ddev-com-front-end.pages.dev/blog/stats-on-ddev-usage-nov-2024/

Contributor training rendered at https://20241103-amplitude-stats.ddev-com-front-end.pages.dev/blog/amplitude-ddev-analytics-contributor-training/

### CODE REVIEW

**Comment by rpkoller** [comment]
thanks for the changes. with the static images there is no more access to third party assets nor the creation of any cookies related to amplitude. from a gdpr perspective the cleaner solution.

File: src/content/blog/stats-on-ddev-usage-nov-2024.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Now that the Traefik router has been standard for two major versions, there's very little usage of the old Nginx Proxy router. We were going to remove it in the upcoming v1.24.0, but we're not having to spend any time with it, so will probably not do that. If you need it, please speak up and let us know.
+
+![Router usage](/img/blog/2024/11/RouterUsage.png)
+
+<a href="https://app.amplitude.com/analytics/share/b443310e41854c4a9c79ff7c5d6b399b" target="_blank">Live version of Router Usage</a>
+
+## Explore them yourself!
+
+You can explore these yourself, ask for permission.** If you're a known community member I'll happily add your access to Amplitude to explore. There are many more things that can be done with this. See [Amplitude Contributor Training](amplitude-ddev-analytics-contributor-training.md).
```
Comment (stasadev) [inline src/content/blog/stats-on-ddev-usage-nov-2024.md:107]:
```suggestion
**You can explore these yourself, ask for permission.** If you're a known community member I'll happily add your access to Amplitude to explore. There are many more things that can be done with this. See [Amplitude Contributor Training](amplitude-ddev-analytics-contributor-training.md).
```

File: src/content/blog/stats-on-ddev-usage-nov-2024.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+summary: DDEV Live Usage Statistics from Amplitude
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/11/ddev-environment-usage.png
+  alt: DDEV Usage of each environment
+categories:
+  - Community
+---
+
+Here's an updated review of some DDEV usage stats. It's always fun to see some of these. Most of these charts are number from the past 7 days. And they are LIVE stats as you view.
```
Comment (stasadev) [inline src/content/blog/stats-on-ddev-usage-nov-2024.md:14]:
```suggestion
Here's an updated review of some DDEV usage stats. It's always fun to see some of these. Most of these charts are number from the past 7 days (Oct 28 - Nov 3, 2024). And check out the LIVE stats link below each chart.
```

**Review by stasadev — CHANGES_REQUESTED** [review]

File: src/content/blog/stats-on-ddev-usage-nov-2024.md (hunk 2 above)
Comment (rfay) [reply src/content/blog/stats-on-ddev-usage-nov-2024.md:14]:
```suggestion
Here's an updated review of some DDEV usage stats. It's always fun to see some of these. Most of these charts are from the past 7 days (Oct 28 - Nov 3, 2024). And check out the LIVE stats link below each chart.
```

**Pull request merged**